src/router/constants.ts

```diff
@@ -27,6 +27,7 @@ export const externalUrls = {
   personopplysninger:
     'https://www.nav.no/personopplysninger-i-pensjonskalkulator',
   spk: 'https://www.spk.no/',
+  klp: 'https://www.klp.no/',
```

Comment:
Når det legges til ny lenke her og i translations.tsx, bør også testen translations.test.tsx utvides med ny test for <klpLink>